app/i18n/messages/sv.json

```
@@ -349,6 +349,9 @@
   "ResourceAvailability.availableAt": "Ledig kl. {time}",
   "ResourceAvailability.availableTime": "Ledig {hours} {hours, plural, one {timme} other {timmar}}",
   "ResourceAvailability.closed": "Stängt",
+  "ResourceAvailability.closedTemporarily": "Closed temporarily __swedish__",
+  "ResourceAvailability.freeUntil": "free until {time} __swedish__",
```

If with free, you mean it doesn't cost anything then change it to this, in other cases can you specify how in what way you mean it's free.
```suggestion
  "ResourceAvailability.freeUntil": "gratis till {time}",
```

---

Free here means more like `available`

---

Then it should be "tillgänglig till {time}"